src/Sulu/Bundle/Sales/CoreBundle/Resources/public/scss/modules/item-table.scss

```diff
@@ -126,6 +126,8 @@ $rowPadding : 0 20px 0 0;
 }
 
 .item-add {
+    width: 41.66%; # grid-col-5
```

Comment:
unfortunately, no, but that shouldn't be a problem since percentage of grid never changes (5 / 12 * 100 = 41.66 )